content/en/blog/2024/collector-roadmap/index.md

```diff
@@ -0,0 +1,85 @@
+
+- [Connected Observability Pipelines in the OpenTelemetry Collector](https://www.youtube.com/watch?v=uPpZ23iu6kI)
+- [Telemetry Showdown: Fluent Bit Vs. OpenTelemetry Collector](https://www.youtube.com/watch?v=ykq1F_3PmJw)
+- [OpenTelemetry Collector Deployment Patterns](https://www.youtube.com/watch?v=WhRrwSHDBFs)
+
+The Collector has been a core component for organizations looking to adopt
+OpenTelemetry as part of their strategy to improve the telemetry emitted by
+their systems. Organizations around the world have already adopted it and
+successfully process large amounts of data through pipelines as documented by
+these various talks:
+
+- [Adopting OpenTelemetry Collector @ eBay - Swapping Engines Mid Flight - Vijay Samuel , eBay](https://www.youtube.com/watch?v=tZJd6W-CIcU)
+- [Ingesting 6.5 Tb of Telemetry Data Daily Through OpenTelemetry Protocol and Coll... Gustavo Pantuza](https://www.youtube.com/watch?v=aDysORX1zIs)
+- [Today, Not Tomorrow: Scalable Strategies for Migrating to Open... Jason Anderson & Kevin Broadbridge](https://www.youtube.com/watch?v=iPGd9_aYu-A)
+- [How and Why You Should Adopt and Expose OSS Interfaces Like OTel... - Daniel Hrabovcak & Shishi Chen](https://www.youtube.com/watch?v=D71fK2MFreI)
+- [Why, How to, and Issues: Tail-Based Sampling in the OpenTelemetry Collector - Reese Lee, New Relic](https://www.youtube.com/watch?v=l4PeclHKl7I)
+
+A few months ago, there was an
+[ask from the community](https://github.com/open-telemetry/community/issues/1971)
+to declare the OpenTelemetry Collector stable.
+
+![Can haz Collector v1?](can-haz-collector.png)
+
+Now you might be asking yourself "Why would anyone want the Collector to be
+declared stable? You just told me it's already used in production!" It's true,
+the Collector and it's configuration has been fairly stable for core components
+for some time. The release of the Collector v1 will signal that the
+OpenTelemetry community is ready to provide long term support and not introduce
+backwards incompatible changes without bumping the major version. This enables
+organizations where the use of the Collector is not possible today due to
+policies around using beta software to consider adopting it. Additionally, it
+helps the community move along some of the work needed that will help its
+graduation process in the CNCF.
+
+The request to stabilize was met with pushback from maintainers since calling
+anything 1.0 has a way of setting expectations indefinitely. This lead to a
+series of discussions and meetings that brought together the maintainers of the
+Collector to decide on what a 1.0 really means for the Collector.
+
+And after a lot of back and forth, we decided on a limited scope of what we
+wanted to focus on, which is a distribution of the Collector that only includes
+an OTLP receiver and an OTLP exporter. It was also decided that individual Go
+modules that the Collector components rely upon must also be marked as stable.
```

Comment:
Should we mention that it means 1.x go api guarantees?

Reply:
Added a link to the versioning guideline